src/vmemcache.h

```diff
@@ -51,6 +51,13 @@ extern "C" {
 #define VMEMCACHE_LEVEL_VAR "VMEMCACHE_LEVEL"
 #define VMEMCACHE_FILE_VAR "VMEMCACHE_FILE"
 
+/*
+ * atomic store lval = rval
+ * 'util_atomic_store_explicit64' does not work correctly with Helgrind and DRD
```

Comment:
it does work correctly, you simply have a race. The fact that it's intentional does not change the fact that fact.